### PR TITLE
crusher compilers: update cce def for cce@14.0.2

### PR DESCRIPTION
**Pull request by eugeneswalker**
Using this in favor of equivalent changes submitted https://github.com/E4S-Project/facility-external-spack-configs/pull/6.

@kwryankrattiger @wspear

### CODE REVIEW

File: crusher/compilers.yaml [hunk 1]
```diff
@@ -29,28 +29,31 @@ compilers:
...
+      f77: ftn
+      fc: ftn
     flags: {}
     operating_system: sles15
     target: any
     modules:
     - PrgEnv-cray/8.3.3
-    - cce/14.0.0
+    - cce/14.0.2
+    - xpmem
```
Comment (kwryankrattiger) [inline crusher/compilers.yaml:44]: just lamenting, but it is crazy we have to load this...

**Review by kwryankrattiger — COMMENTED** [review]
Just one comment, I am going to test it quick.

File: crusher/compilers.yaml [hunk 2]
```diff
@@ -29,28 +29,31 @@ compilers:
...
     - /opt/cray/gcc-libs
 - compiler:
-    spec: cce@14.0.0
+    spec: cce@14.0.2
     paths:
-      cc: /opt/cray/pe/craype/2.7.13/bin/cc
-      cxx: /opt/cray/pe/craype/2.7.13/bin/CC
-      f77: /opt/cray/pe/craype/2.7.13/bin/ftn
-      fc: /opt/cray/pe/craype/2.7.13/bin/ftn
+      cc: cc
```
[the remark this answers is not in the document]
Comment (eugeneswalker) [reply crusher/compilers.yaml:34]: I did the whole E4S 22.11 stack without absolute paths. I'm fine assuming this is OK for now in order to save us one more set of version specific things we have to keep up to date. What do you think?

File: crusher/compilers.yaml (hunk 1 above)
Comment (eugeneswalker) [reply crusher/compilers.yaml:44]: I agree.

File: crusher/compilers.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kwryankrattiger) [reply crusher/compilers.yaml:34]: just tested it as well an no problems either. I am okay to remove it.

**Comment by kwryankrattiger** [comment]
+2 Looks good

**Pull request merged**